src/components/ScholarshipListCard.jsx

```diff
@@ -24,19 +29,31 @@ function ScholarshipListCard({ scholarship }) {
           state: { scholarship },
         }}>
         <CardContent sx={{ p: 3 }}>
-          <Typography variant="body1" gutterBottom sx={{ fontWeight: 'bold' }}>
-            {deadline?.toLocaleDateString()}
-          </Typography>
-          <Typography variant="h5">{name}</Typography>
           <Typography variant="subtitle1" sx={{ fontWeight: '500' }}>
             {organization}
           </Typography>
-          <Typography
-            variant="subtitle1"
-            sx={{ fontWeight: '500' }}
-            gutterBottom>
-            {ScholarshipAmount.toString(amount)}
+          <Typography variant="h5" gutterBottom>
```

Comment:
As the two cards become more like each other I wonder if there's a way to logically combine them or break the top part of them apart so we don't need to have duplicate code in multiple places.

Probably an exercise for another PR.